zipkin-server/src/main/resources/zipkin-server-shared.yml

```diff
@@ -82,7 +82,7 @@ zipkin:
       max-queue-size: ${STORAGE_THROTTLE_MAX_QUEUE_SIZE:1000}
     mem:
       # Maximum number of spans to keep in memory.  When exceeded, oldest traces (and their spans) will be purged.
-      max-spans: 500000
+      max-spans: ${STORAGE_MEM_MAX_SPANS:500000}
```

Comment:
given that we have `CASSANDRA`, `ES` and `MYSQL` as prefixes for the other storage options maybe `INMEMORY_MAX_SPANS` ?

Reply:
PTAL at last commit